conda/base/context.py

```diff
@@ -969,18 +969,10 @@ def use_only_tar_bz2(self):
         #    is only called when use_only_tar_bz2 is first called.
         import conda_package_handling.api
 
-        use_only_tar_bz2 = False
-        if self._use_only_tar_bz2 is None:
-            if self._argparse_args and "use_only_tar_bz2" in self._argparse_args:
-                use_only_tar_bz2 &= self._argparse_args["use_only_tar_bz2"]
         return (
-            (
-                hasattr(conda_package_handling.api, "libarchive_enabled")
-                and not conda_package_handling.api.libarchive_enabled
-            )
-            or self._use_only_tar_bz2
-            or use_only_tar_bz2
-        )
+            hasattr(conda_package_handling.api, "libarchive_enabled")
+            and not conda_package_handling.api.libarchive_enabled
```

Comment:
```suggestion
            not getattr(conda_package_handling.api, "libarchive_enabled", False)
```

Reply:
@dholth thoughts on simplifying it this way? I've also checked and `conda_package_handling.api.libarchive_enabled` has been available since `CPH 1.4.0` (https://github.com/conda/conda-package-handling/commit/ab7bf9cc5f9e62a2829db81fd37820fb4c5c4677) so since we depend on `CPH>=2.2.0` the constant should always be defined

Reply:
getattr is better but `not conda_package_handling.api.libarchive_enabled` is best.